requirements/constraints.txt

```diff
@@ -14,5 +14,7 @@ Django<2.0
 # TODO: Add contraint explanation
 pylint==1.9.3
 pycodestyle==2.5.0
+django-fake-model==0.1.4
```

Comment:
Can you sort these contraints? And also please add a comment before each constraint in order to identify why that constraint was added

Reply:
Yes! I'm on it! Thank you

Reply:
Done. Is it better now?